packages/web/src/components/withCellHOC.tsx

```diff
@@ -119,19 +129,30 @@ export const withCell = ({
         {...beforeQuery(variables)}
       >
         {({ error, loading, data, ...queryRest }) => {
+          const processedData = data && afterQuery(data)
+
           if (error) {
             if (Failure) {
               return <Failure error={error} {...queryRest} {...props} />
             } else {
               throw error
             }
           } else if (loading) {
-            return <Loading {...queryRest} {...props} />
+            return isEmpty(processedData) ? (
+              <Loading {...queryRest} {...props} />
+            ) : (
+              <Success
+                {...afterQuery(data)}
+                {...queryRest}
+                {...props}
+                loading={true}
+              />
+            )
```

Comment:
Is there no way we can do this without this being a breaking change? What if I actually want `<Loading>` while it's fetching, even if I already have data?

Reply:
A flag could be introduced, the where is a bigger question. It'd also be a valuable discussion to figure out if the default would be to show `Loading` or `Success - loading`.

Reply:
As I mentioned elsewhere another option is to pass all data to <Loading>, and then leave it up to the user if they want to reuse the same code for <Loading> and <Success> or not. That would be backwards compatible. But maybe not the prettiest implementation on their side.

Reply:
That is an option, and it makes sense too. It would interfere with the idea of using `Suspense` internally, since `Suspense` won't be triggered in stale updates, afaik.